testbed/tests/test_images.py

```diff
@@ -16,14 +17,21 @@ def image_probe(app, image):
 
 
 async def test_local_image(app):
-    "An image can be specified by filename"
+    """An image can be specified by filename"""
     image = toga.Image("resources/sample.png")
     assert image.width == 144
     assert image.height == 72
 
 
+async def test_closed_file_handle(app):
+    """The local image file isn't left open once the Image is created."""
+    _ = toga.Image("resources/sample.png")
+    # If the file still has an open handle, this should raise an IOError.
+    _ = Path(app.paths.app / "resources/sample.png").read_bytes()
```

Comment:
pardon the interjection....but reading the file should be fine; however, deleting the file here should at least cause problems on Windows.